src/Mapbender/ManagerBundle/Resources/public/js/application-edit.js

```diff
@@ -324,6 +319,31 @@ $(function() {
         return false;
     });
 
+    function elementFormSubmit() {
+        var form = $("#elementForm"),
+            data = form.serialize(),
+            url = form.attr('action'),
+            self = this;
+
+        $.ajax({
+            url: url,
+            method: 'POST',
+            data: data,
+            error: function (e, statusCode, message) {
+                alert(Mapbender.trans("mb.application.save.failure.general") + ' ' + message);
```

Comment:
prevent using `alert()` in JavaScript! Don't we have a better way to display errors in Mapbender?

I mean there was something like: `Mapbender.error()`...